### ISSUE TITLE
L2TP-VPN Server did not respond

### ISSUE THREAD

**Issue by ianoroceo** [webdigi/AWS-VPN-Server-Setup#44]
![network_and_slack_-_fox_mediacloud](https://user-images.githubusercontent.com/15149237/32570599-76b6c87e-c479-11e7-8d16-a10198d4e36c.jpg)

I am getting this error after creating the Stack.
on the Template Json it was mentioned that I have to wait for 5minutes

> Description": "Allow upto 5 minutes after setup to connect. Please contact us at https://www.webdigi.co.uk/blog/2015/how-to-setup-your-own-private-secure-free-vpn-on-the-amazon-aws-cloud-in-10-minutes/ for any help!

is there a configuration that I have to change on that template?

**Comment by rosoposo** [comment]
Did you wait for 5 minutes and try again? Also try a little longer. Sorry try another region is that does not work

**Comment by ianoroceo** [comment]
went to lunch for one hour and retried connecting and still got the same error

created another stack using different Region. waited for 20mins before trying to connect. still the same error

![network_and_slack_-_fox_mediacloud_sao paulo](https://user-images.githubusercontent.com/15149237/32576013-8146015e-c48a-11e7-9aaa-0605841f7248.jpg)

**Comment by webdigi** [comment]
we had reports like this previously and most issues were caused due to the host network blocking VPN connections. Please try connecting to the same server using your mobile with mobile network and you should see they work.

**Comment by ianoroceo** [comment]
Yes, I was able to connect on one of the VPN I created after connecting my machine to a public wifi.
I also observed that it won’t work if the password and secret is more than 4 chars. It says minimum is 4 but when I tried setting it to 8 numeric characters, it won’t work until bring it back to 4 numeric characters

 Ian Christopher O. Oroceo

Cell: (213) 248-0189
Quality Assurance Engineer

[http://mediacloud.fox/wp-content/uploads/2017/01/fmc-h-color-300px-email-sig.png]<http://foxemf.com/FMCEmailTags>
